### ISSUE TITLE
Fix R Tests

### ISSUE THREAD

**Issue by hannes** [duckdb/duckdb#4549]
```
Component "lgl": 'is.NA' value mismatch: 1 in current 1 in target
84
  ── Failure (test_arrow.R:74:3): to_duckdb ──────────────────────────────────────
85
  ... %>% arrange(sum_int) not equal to ... %>% arrange(sum_int).
86
  Component "lgl": 'is.NA' value mismatch: 1 in current 1 in target
[87](https://github.com/duckdb/duckdb/runs/8100189704?check_suite_focus=true#step:9:88)
  
[88](https://github.com/duckdb/duckdb/runs/8100189704?check_suite_focus=true#step:9:89)
```

Yay arrow tests

**Comment by pdet** [comment]
@hannes I fixed this already in #4414 

(Classic pushing small fixes to unrelated prs, sorry)

**Comment by hannes** [comment]
I've opened a separate PR to merge this before the release to not derail deployment
